### ISSUE TITLE
[Feature]: Range Numeric data type in classification store

### ISSUE THREAD

**Issue by zekuraz** [pimcore/pimcore#15944]
### Feature description

In the last version of Pimcore, you add the numericRange object field, consists of two numeric fields (minimum and maximum) and so can be configured the same way. It's work like a charm, but not supported in classification store.

**Comment by ITspirit** [comment]
Is there any planned progress on this topic @fashxp @brusch ?

It´s a valid point and tbh. the Quantity Value Range is more important within the classification store as in a simple DataObject.

I have the need for it in, so before i start working on a PR just asking, because ticket is already 5 month old.

**Comment by fashxp** [comment]
I agree, range data types definitely make sense in classification store too. It is not on our short-term roadmap though. 
But we are ready merge PRs for it.
